src/content/reference/app-defaulting-validation/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Defaulting and Validation of App CRs
+description: How defaulting and validation of app CRs is implemented by app-admission-controller
+weight: 110
+type: page
+last_review_date: 2020-12-16
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+---
+
+## Overview
+
+For Giant Swarm releases using app-operator version 3.0.0 and upwards the
```

Comment:
```suggestion
For Giant Swarm releases using app-operator version 3.0.0 and upwards (As of writing: Azure v13.1.0+, AWS and KVM TBD) the
```

Would it be helpful to specify which workload cluster releases has this as of today? Customers and SEs think in terms of those, right?

Reply:
Yes at the moment its just Azure >= v13.1.1 there are no AWS and KVM releases.

This is why it explains how to check if the release has app-operator v3.0.0. But I agree its easier for customers and SEs to state the versions. We just need to remember to update this.

 I'll add something.

Reply:
@cokiengchiara I added a note for this.

Reply:
perfect!